app/views/exception_hunter/errors/index.html.erb

```diff
@@ -25,7 +25,7 @@
 <% @errors.each do |error| %>
   <div class="row error-row">
     <div class="column column-75 error-cell">
-      <a href="#"><%= error.message %></a>
+      <a href="errors/<%= error.id %>"> <%= error.message %></a>
```

Comment:
You can use a link helper for this

Reply:
couldn't find any

Reply:
```suggestion
      <%= link error.message, to: error_path(error) %>
```

Or something like that

Reply:
ooh I did not just missed them helpers 😆